charts/mailhog/templates/deployment.yaml

```diff
@@ -96,15 +97,16 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- if .Values.auth.enabled }}
+      {{ if or .Values.auth.enabled .Values.outgoingSMTP.enabled }}
       volumes:
+      {{ end }}
```

Comment:
Please move this `{{ end }}` to the end of the volumes block. This works as well but the proposed change is more clear. I would have added it to the PR myself bu you did not allow PR edits...